test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -646,6 +646,10 @@ Describe "PSModulePath changes interacting with other PowerShell processes" -Tag
             $errors | Should -Be $null
         }
 
+        It "Allows Windows PowerShell subprocesses to load WinPS version of `$PSHOME modules" {
```

Comment:
It seems like, given the scenario outlined in the PR, we should ideally have a test that uses a pwsh install (like maybe one from a zip)

Reply:
In fact in CI system we have exactly that - the parent PS comes with OS in `c:\program files\powershell\7` and PS that is running the test comes from the build directory.
Also some other WinCompat tests will fail without this change so they are also used as tests for this functionality.